src/Analysis/GWASApp/Components/Diagrams/PhenotypeHistogram/PhenotypeHistogram.css

```diff
@@ -1,9 +1,12 @@
 .GWASUI-column.transformation-dropdown-label {
```

Comment:
In the input view the labels and values are aligned for the minimum and maxium outlier cutoffs, but in the modal they're displayed on separate lines. Do we want to style these so they match? 

Version in Modal : 
<img width="639" alt="image" src="https://github.com/user-attachments/assets/038abda5-adc4-4fe2-ac34-6935a8308e58" />
Version for Input: 
<img width="590" alt="image" src="https://github.com/user-attachments/assets/d9a35545-99be-4442-9cac-889db450fe42" />

Reply:
yes, I agree we want them to match 👍

Reply:
I got the modal version to render closer to the one in the input view and pushed a commit: 

After: 
![image](https://github.com/user-attachments/assets/00f65a0d-a217-49ac-9a6a-1d8d0db681bd)